yarn-project/prover-node/src/prover-node.ts

```diff
@@ -86,6 +82,13 @@ export class ProverNode implements ClaimsMonitorHandler, EpochMonitorHandler {
     } catch (err) {
       this.log.error(`Error handling claim for epoch ${proofClaim.epochToProve}`, err);
     }
+
+    try {
+      // Staked amounts are lowered after a claim, so this is a good time for doing a top-up if needed
```

Comment:
It doesn't really hurt, but this one should be unnecessary I think. 

It seems what will happen if people set their min/target to the same number, then if their claim gets picked they will immediately top-up, but then when the proof lands they'll be above the target. 

I think the one check you do before submitting the quote should be sufficient.

Reply:
True, but I wanted to avoid blocking sending a quote because of having to top-up. Immediately after a claim seemed like a good moment where nothing pressing should be happening, so they can look into it. And if they end up above target, it's not too bad.

Reply:
> It seems what will happen if people set their min/target to the same number,

One option could be **not** allowing them to set both numbers, but rather request only one and compute the other one, to make sure they are not equal.